modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -238,6 +238,23 @@ The column is a `STRING` value specifying a replacement function/procedure if th
 | Feature
 | Details
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+GRANT READ {*} ON GRAPH * FOR (n) WHERE n.securityLevel > 3 TO regularUsers
+----
+[source, cypher, role=noheader]
+----
+GRANT TRAVERSE ON GRAPH * FOR (n:Email) WHERE n.classification IS NULL TO regularUsers
+----
+[source, cypher, role=noheader]
+----
+DENY MATCH {*} ON GRAPH * FOR (n) WHERE n.classification <> 'UNCLASSIFIED' TO regularUsers
+----
+| Introduction of link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/property-based-access-control.adoc[property-based access control] for read privileges. The ability to read, traverse and match nodes based on node property values is now supported in Enterprise Edition.
```

Comment:
I'm not sure if the link is working, it does not work when I use it in the published version from this PR.
https://neo4j-docs-cypher-1046.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility/#cypher-deprecations-additions-removals-5.21

Reply:
thanks. I needed to remove `.adoc`. Done now.